test/integration/Protocol.t.sol

```diff
@@ -24,28 +24,29 @@ contract ProtocolIntegrationTests is TestBase {
 
     uint256 internal _excess;
 
-    function setUp() public override {
-        super.setUp();
-
-        _addToList(_EARNERS_LIST, address(_wrappedMToken));
+    function setUp() external {
         _addToList(_EARNERS_LIST, _alice);
         _addToList(_EARNERS_LIST, _bob);
 
-        _wrappedMToken.enableEarning();
-
         _wrappedMToken.startEarningFor(_alice);
         _wrappedMToken.startEarningFor(_bob);
 
         _totalEarningSupplyOfM = _mToken.totalEarningSupply();
     }
 
-    function test_initialState() external view {
+    function test_initialState() external {
+        // TODO: Reinstate to test post-migration for new version.
+        vm.skip(true);
```

Comment:
what is the issue with current version?

Reply:
The protocol tests are to check that the implementation meets mathematical expectations. Now that it is deployed, there is no point to update all the literal values in thee tests just yet. When v2 is finalized, we can make one pass to enable and fix these tests.